frontend/src/app/account/history/history.component.html

```diff
@@ -0,0 +1,46 @@
+<div class="table-container">
+    <p-table
```

Comment:
According to our Milestone 1, 

> The system shall allow the user to sort the history after filtering. 

We could use this? https://primeng.org/table#sort

Reply:
Sure thing